dashboard/client/src/components/StatusChip.tsx

```diff
@@ -23,71 +21,69 @@ import {
 } from "../type/serve";
 import { TypeTaskStatus } from "../type/task";
 
+const orange = "#DB6D00";
+
 const colorMap = {
   node: {
     ALIVE: green,
     DEAD: red,
   },
   worker: {
     ALIVE: green,
+    DEAD: red,
   },
   actor: {
     [ActorEnum.ALIVE]: green,
     [ActorEnum.DEAD]: red,
-    [ActorEnum.PENDING]: blue,
-    [ActorEnum.RECONSTRUCTING]: lightBlue,
+    [ActorEnum.DEPENDENCIES_UNREADY]: orange,
+    [ActorEnum.PENDING_CREATION]: orange,
+    [ActorEnum.RESTARTING]: orange,
   },
   task: {
     [TypeTaskStatus.FAILED]: red,
     [TypeTaskStatus.FINISHED]: green,
     [TypeTaskStatus.RUNNING]: blue,
-    [TypeTaskStatus.RUNNING_IN_RAY_GET]: blue,
-    [TypeTaskStatus.RUNNING_IN_RAY_WAIT]: blue,
-    [TypeTaskStatus.SUBMITTED_TO_WORKER]: "#cfcf08",
-    [TypeTaskStatus.PENDING_ARGS_FETCH]: blue,
-    [TypeTaskStatus.PENDING_OBJ_STORE_MEM_AVAIL]: blue,
-    [TypeTaskStatus.PENDING_NODE_ASSIGNMENT]: "#cfcf08",
-    [TypeTaskStatus.PENDING_ARGS_AVAIL]: "#f79e02",
+    [TypeTaskStatus.SUBMITTED_TO_WORKER]: orange,
+    [TypeTaskStatus.PENDING_NODE_ASSIGNMENT]: orange,
+    [TypeTaskStatus.PENDING_ARGS_AVAIL]: orange,
+    [TypeTaskStatus.NIL]: grey,
```

Comment:
The grey doesn't seem right. [The spec ](https://www.figma.com/file/1BaUL5zyv6rlnOo47Iio4u/%5Bvision%5D-Anyscale%2FRay-Observability?node-id=3394%3A147769&t=m6slo2y0g93pzK1K-1)used #5F6469

Reply:
<img width="1401" alt="image" src="https://github.com/ray-project/ray/assets/125417081/b2fd1bbd-1e02-4284-88a5-95abb71df5a6">
<img width="1461" alt="image" src="https://github.com/ray-project/ray/assets/125417081/8b9dafbc-85f4-4b91-9c7b-1076141dc4c6">